app/main/views/buyers.py

```diff
@@ -141,13 +152,15 @@ def update_brief(brief_id):
         flash(e.message, 'error')
         brief = data_api_client.get_brief(brief_id).get('briefs')
         users = brief.get('users')
+        sellers = brief.get('sellers')
         title = brief.get('title')
         return render_template_with_csrf(
             "view_buyers.html",
             users=users,
             title=title,
             brief_id=brief_id,
-            brief=brief
+            brief=brief,
+            sellers=sellers
```

Comment:
Do you need to bring this in any more? It seems like the `view_buyers.html` template is always referencing `brief.sellers` and not this param.

Reply:
You're correct, I didn't need that anymore. I will remove it, cheers.